lisa/transformers/kernel_installer.py

```diff
@@ -212,6 +212,7 @@ def install(self) -> str:
             f"deb {self.repo_url} {version_name} "
             f"restricted main multiverse universe"
         )
+        ubuntu.wait_cloud_init_finish()
```

Comment:
Should it be added into `wait_running_package_process`?

Reply:
then it will be run in every package installation.

Reply:
That's what we need to think about, why is the kernel installer special to wait cloud init? Why don't others need? Are we wrapping the functions in right way?

Reply:
kernel installer will update sources. List file by adding repo, if cloud-init does not finish the initialization work, the previous change will be overwritten.

Reply:
the os supports `add_repository`, can this be changed to use that method?

Reply:
updated